Cargo.toml

```diff
@@ -111,6 +111,7 @@ rpath = true
 [patch.crates-io]
 prost = { git = "https://github.com/danburkert/prost", rev = "6113789f70b69709820becba4242824b4fb3ffec" }
 prost-derive = { git = "https://github.com/danburkert/prost", rev = "6113789f70b69709820becba4242824b4fb3ffec" }
+slog = { git = "https://github.com/garbageslam/slog", rev = "cc1562258b81353426d17a55b7447256fa6eabbf" }
```

Comment:
Please include a comment explaining why we are patching this. I'd like us to move to a pattern where it is more obvious just by looking at this file, versus having to go and decipher what mandated that from the commit log.

Reply:
yeah okay

Reply:
done